app/controllers/articles_controller.rb

```diff
@@ -16,13 +16,19 @@ def create
     respond_with article, action: :index
   end
 
+  def edit; end
+
+  def update
```

Comment:
What do you think about moving this logic to `Articles::PublicationsController#create`?